x-pack/plugins/watcher/public/sections/watch_list/components/watch_list.tsx

```diff
@@ -52,6 +55,56 @@ const WatchListUi = ({ intl }: { intl: InjectedIntl }) => {
     [watches, deletedWatches]
   );
 
+  const createWatchButtons = (
+    <EuiFlexGroup gutterSize="m">
+      <EuiFlexItem grow={false}>
+        <EuiToolTip
+          position="top"
+          content={
+            <FormattedMessage
+              id="xpack.watcher.sections.watchList.createThresholdAlertButtonTooltip"
+              defaultMessage="Send an alert on a specific condition"
+            />
+          }
+        >
+          <EuiButton
+            data-test-subj="createThresholdAlertButton"
+            iconType="plusInCircle"
+            href="#/management/elasticsearch/watcher/watches/new-watch/threshold"
+          >
+            <FormattedMessage
+              id="xpack.watcher.sections.watchList.createThresholdAlertButtonLabel"
+              defaultMessage="Create threshold alert"
+            />
+          </EuiButton>
+        </EuiToolTip>
+      </EuiFlexItem>
+
+      <EuiFlexItem grow={false}>
+        <EuiToolTip
+          position="top"
+          content={
+            <FormattedMessage
+              id="xpack.watcher.sections.watchList.createAdvancedWatchTooltip"
+              defaultMessage="Set up a custom watch in raw JSON"
+            />
+          }
+        >
+          <EuiButton
+            data-test-subj="createAdvancedWatchButton"
+            iconType="plusInCircle"
+            href="#/management/elasticsearch/watcher/watches/new-watch/json"
+          >
+            <FormattedMessage
+              id="xpack.watcher.sections.watchList.createAdvancedWatchButtonLabel"
+              defaultMessage="Create advanced watch"
+            />
+          </EuiButton>
+        </EuiToolTip>
+      </EuiFlexItem>
+    </EuiFlexGroup>
+  );
+
   if (getPageErrorCode(error)) {
```

Comment:
I think this is only checking for `403` and `404` errors. Do we need to guard against other status codes?

Reply:
Yes I think it's probably best to render a danger callout with the error status and message in the event of non-403 and non-404 errors, since if there's an error then the table can't render content and might throw a fatal error (e.g. by trying to access a property on an undefined variable). I think a callout is better than a toast notification in this case because a toast notification is transient and the user might want to report the error in an issue or something. WDYT?

Reply:
yeah, makes sense - updated.

<img width="1162" alt="Screen Shot 2019-05-23 at 10 02 26 AM" src="https://user-images.githubusercontent.com/5226211/58259678-0021b280-7d43-11e9-9c59-de2ed7b0beb1.png">